source/HailPanel.cpp

```diff
@@ -55,25 +55,18 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship)
 		message = "(There is no response to your hail.)";
 	else if(!hasLanguage)
 		message = "(An alien voice says something in a language you do not recognize.)";
-	else if(gov->IsEnemy())
+	else if(gov->IsEnemy() && !ship->IsDisabled())
```

Comment:
Right here you could add
`// Update default hail responses based on the hailed ship's status conditions and government.`

Also, is there any repercussion from allowing hostile ships to fall down to the `else` condition? In the original, only friendly ships can make it that far